plugin/src/main/resources/seedJobGroovyScript.groovy

```diff
@@ -133,12 +133,14 @@ def localXmlBase = new FilePath(localFile)
 println 'Copy remote files to local XML dir'
 xmlBase.copyRecursiveTo(localXmlBase)
 
```

Comment:
The reason in the commit message is not really valid with the generated name from the commit before, maybe just finish it after "happens", because the first sentence is reason enough :)

Reply:
Ok right, I'm still figuring out how GitHub differs from Gerrit :-).

Commit fix done.